content/04.experiment-types.md

```diff
@@ -16,4 +16,146 @@ Proteomics experiments can be both qualitative and quantitative.
+AP-MS has been used to study a wide range of biological processes, including signal transduction pathways, protein complexes, and protein post-translational modifications. 
+Despite its widespread use, AP-MS has some limitations, including the potential for non-specific interactions, the difficulty in interpreting complex data sets, and the possibility of missing important interacting partners due to limitations in sensitivity or specificity. 
+However, with continued advances in technology and data analysis methods, AP-MS is likely to remain a valuable tool for the study of protein-protein interactions and for the development of new therapeutics.
+
+2. Avidity-based Extracellular Interaction Screen (APEX) [@doi:10.1186/1471-2105-9-529; @doi:10.1007/978-1-4939-9537-0_4; @doi:10.1155/2010/840518]
+Avidity-based Extracellular Interaction Screen (APEX) is a proximity labeling technique that allows for the identification of cell surface protein interactions in living cells. 
+By labeling cell surface proteins, APEX enables the identification and characterization of protein interactions that occur on the extracellular side of the cell membrane.
+One of the major advantages of APEX is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions. 
+In addition, APEX is able to label proteins on the extracellular side of the cell membrane, allowing for the identification of cell surface protein interactions. APEX has been used to identify a wide range of protein interactions, including receptor-ligand interactions, signaling complexes, and extracellular matrix interactions.
+Despite its advantages, APEX has some limitations, including the potential for non-specific labeling, the difficulty in distinguishing between direct and indirect interactions, and the possibility of missing interactions that occur at low abundance or in regions of the cell that are not effectively labeled. 
+
+3. Biochemical Identification of Neighbors by proximity Labeling (BioID) [@doi: 10.1074/mcp.R120.001941; @doi:10.1002/pmic.201600123; @doi:10.1091/mbc.E15-12-0844]
+Biochemical Identification of Neighbors by proximity Labeling (BioID) is a proximity labeling technique that allows for the identification of protein-protein interactions. 
+BioID involves the introduction of a genetically-encoded biotin ligase enzyme into cells, which biotinylates proteins in close proximity to the enzyme. 
+By labeling proteins that are in close proximity to the biotin ligase, BioID enables the identification and characterization of protein interactions that occur in living cells.
+One of the advantages of BioID is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions. In addition, BioID is able to label proteins that are not easily accessible by other proximity labeling techniques, such as APEX. 
```

Comment:
```suggestion
One of the advantages of BioID is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions. 
In addition, BioID is able to label proteins that are not easily accessible by other proximity labeling techniques, such as APEX. 
```
Can you add why bioID can label proteins that are not accessible by APEX? The reason is not immediately obvious to me